settlement/dymension/types.go

```diff
@@ -0,0 +1,10 @@
+package dymension
+
+import (
+	dymension "github.com/dymensionxyz/dymension/v3/x/rollapp/types"
+)
+
+var (
+	// if this error is returned, the batch is already submitted
+	ErrBatchAlreadySubmitted = dymension.ErrWrongBlockHeight.Error()
+)
```

Comment:
instead of .Error() and string compare
wrap it and use errorsmod.Isof

Reply:
I got the hub's error by `txResp, err := c.cosmosClient.BroadcastTx` so not sure how I can check the error w/o string compare

Reply:
ah I see, I didnt' understand that first time
Yes you are right, you have to use str compare
but it should be encapsulated, I made a patch